share/module_check_a_mundo.F

```diff
@@ -569,6 +570,27 @@ END FUNCTION bep_bem_nbui_max
       END DO
 
 #if (EM_CORE == 1)
+!--------------------------------------------------------------------------------------------------
+! Input tables must exist in running directory for fast bin microphysics scheme (mp_physics = 30)
+!--------------------------------------------------------------------------------------------------
+      IF (  model_config_rec % mp_physics(1) .EQ. FAST_KHAIN_LYNN_SHPUND   ) THEN
+           INQUIRE(FILE='./SBM_input_33', EXIST=fsbm_table1_exists)
+           IF (.not.fsbm_table1_exists ) THEN
+           wrf_err_message = "Input table SBM_input_33 doesn't exist !!!"
+           CALL wrf_debug ( 0, wrf_err_message )
+           wrf_err_message = 'Download this table from http://www2.mmm.ucar.edu/wrf/src/wrf_files/'
+           CALL wrf_debug ( 0, wrf_err_message )
+           count_fatal_error = count_fatal_error + 1
```

Comment:
@smileMchen 
Ming,
Indent these lines and the similar lines in the next IF test.

Reply:
@davegill 
Dave,
I have changed the indent as suggested. Thanks.